simulator/writeback/writeback.cpp

```diff
@@ -74,6 +74,18 @@ void Writeback<ISA>::check( const FuncInstr& instr)
 }
 
 #include <mips/mips.h>
+#include <risc_v/risc_v.h>
 
 template class Writeback<MIPS>;
+template class Writeback<RISCV32>;
+template class Writeback<RISCV64>;
+template class Writeback<RISCV128>;
+
+template class InstrMemory<RISCVInstr<uint32>>;
```

Comment:
Why do we need these in every file? These are header-only classes, so they are instantied implicitly